modules/travel_pay/app/controllers/travel_pay/claims_controller.rb

```diff
@@ -2,21 +2,6 @@
 
 module TravelPay
   class ClaimsController < ApplicationController
-    ##
-    # For now, index is integrated with PING endpoint until
-    # upstream API is more complete.
-    def index
-      veis_response = client.request_veis_token
-
-      veis_token = veis_response.body['access_token']
-
-      btsss_ping_response = client.ping(veis_token)
-
-      render json: { data: "Received ping from upstream server with status #{btsss_ping_response.status}." }
-    end
-
-    def client
-      TravelPay::Client.new
-    end
+    def index; end
```

Comment:
I think we should just get rid of this controller since there's just an empty index method in it. Easy enough to create again if we need.